src/shared/services/document-management/document-management.service.ts

```diff
@@ -26,4 +30,38 @@ export class DocumentManagementService {
         map(response => response.json())
       );
   }
+
+  getMediaViewerInfo(caseFieldValue: any): string {
```

Comment:
can we rename `caseFieldValue` to `documentFieldValue`?

Reply:
Done.